app/src/main/java/org/projectbuendia/client/providers/Contracts.java

```diff
@@ -36,7 +36,8 @@
         ORDERS("orders"),
         PATIENTS("patients"),
         USERS("users"),
-        SYNC_TOKENS("sync_tokens");
+        SYNC_TOKENS("sync_tokens"),
+        UNSYNC_TOKENS("unsync_forms");
```

Comment:
(2) is a good idea. Additionally, we also try to send any pending data with every sync, which, while in the patient chart, is every minute.

Regarding (1), I think you're right - we need to explicitly call out and explain this.

We don't have any design notes about this except for what we've got in [Pivotal Tracker](https://www.pivotaltracker.com/story/show/107200344).